src/ResourceManager/Network/Commands.Network/help/New-AzureRmRouteTable.md

```diff
@@ -15,7 +15,7 @@ Creates a route table.
 
 ```
 New-AzureRmRouteTable -ResourceGroupName <String> -Name <String> -Location <String> [-Tag <Hashtable>]
- [-Route <System.Collections.Generic.List`1[Microsoft.Azure.Commands.Network.Models.PSRoute]>] [-Force]
+ [-Route <System.Collections.Generic.List`1[Microsoft.Azure.Commands.Network.Models.PSRoute]>] [-DisableBgpRoutePropagation <Boolean>] [-Force]
```

Comment:
This shouldn't be formatted as a boolean - it should be just [-DisableBgpRoutePropagation].  Can you regenerate this file and see if this issue still exists?